xml_converter/attribute_test_script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ATTRIBUTE_TEST_CASES_FOLDER_PATHS=("./test_cases/can_fade_tests")
+
+ATTRIBUTE_TEST_CASES=()
```

Comment:
You dont do anything with this variable in the global scope, and you never reset it within the primary loop. So each iteration of the primary loop you will end up reiterating over this variable which contains all the previous values of previous iterations. If you refactor to remove this variable completely things will be cleaner.